src/structured-data/structured-data-tags.js

```diff
@@ -92,6 +92,7 @@ function generateArticleData (structuredData = {}, story = {}, publisherConfig =
   return Object.assign({}, generateCommonData(structuredData, story, publisherConfig), {
     "author": authorData(authors),
     "keywords": metaKeywords.join(','),
+    "thumbnailUrl": imageUrl(publisherConfig, story['hero-image-s3-key']),
```

Comment:
Shouldn't this be of a smaller size? This will give you the proper big hero image right?

Please check if Google has recommendations on thumbnail size or look at other publishers like Guardian on what size they are using.

Reply:
Knocksense using a 210 width image. `imageUrl` will be giving a 480x270 image. I think that size is fine.